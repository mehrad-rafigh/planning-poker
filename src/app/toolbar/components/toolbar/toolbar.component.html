<nav class="navbar d-flex row-no-wrap">
  <span class="navbar-title f-start">Planning Poker</span>
</nav>
