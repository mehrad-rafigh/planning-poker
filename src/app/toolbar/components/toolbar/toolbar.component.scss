:host {
  margin-bottom: 4rem;

  .navbar {
    width: 100%;
    height: 68px;
    background-color: lightblue;

    &-title {
      margin: auto;
      font-size: 22px;
      font-weight: bold;
    }
  }
}
