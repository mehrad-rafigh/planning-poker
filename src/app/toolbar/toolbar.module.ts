import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ToolbarComponent } from './components/toolbar/toolbar.component';
import { MatToolbarModule } from "@angular/material";



@NgModule({
  declarations: [ToolbarComponent],
  imports: [
    CommonModule,
    MatToolbarModule
  ],
  exports: [ToolbarComponent]
})
export class ToolbarModule { }
