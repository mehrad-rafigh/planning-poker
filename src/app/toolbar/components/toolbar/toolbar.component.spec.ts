import { async, ComponentFixture, TestBed } from "@angular/core/testing";

import { ToolbarComponent } from "./toolbar.component";
import { DebugElement } from "@angular/core";
import { By } from "@angular/platform-browser";

describe("ToolbarComponent", () => {
  let component: ToolbarComponent;
  let fixture: ComponentFixture<ToolbarComponent>;
  let debugElement: DebugElement;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ToolbarComponent]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ToolbarComponent);
    component = fixture.componentInstance;
    debugElement = fixture.debugElement;
    fixture.detectChanges();
  });

  it("should create", () => {
    expect(component).toBeTruthy();
  });

  it("should match the snapshot", () => {
    expect(fixture).toMatchSnapshot();
  });

  it("should display the app name", () => {
    const span: HTMLSpanElement = debugElement.query(By.css("span"))
      .nativeElement;
    expect(span.innerHTML).toBe("Planning Poker");
  });
});
