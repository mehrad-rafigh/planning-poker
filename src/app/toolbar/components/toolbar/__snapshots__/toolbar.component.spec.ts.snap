// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarComponent should match the snapshot 1`] = `
<app-toolbar>
  <nav
    class="navbar d-flex row-no-wrap"
  >
    <span
      class="navbar-title f-start"
    >
      Planning Poker
    </span>
  </nav>
</app-toolbar>
`;
